### PR TITLE
Kubectl context message

### PR DESCRIPTION
**Pull request by chrislovecnm**
Closes https://github.com/kubernetes/kops/issues/397, https://github.com/kubernetes/kops/issues/1000

Please review and merge.

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kops/1050)

### CODE REVIEW

**Comment by chrislovecnm** [comment]
Good to go 😀

**Comment by chrislovecnm** [comment]
@k8s-bot aws e2e test this

**Comment by kris-nova** [comment]
LGTM - simple change, and some much needed tests

File: upup/pkg/kutil/kubecfg_builder.go [hunk 1]
```diff
@@ -183,20 +184,31 @@ func (c *KubeconfigBuilder) WriteKubecfg() error {
 			return err
 		}
 	}
+	fmt.Printf("Wrote config for %s to %q\n", c.Context, c.KubeconfigPath)
```
Comment (justinsb) [inline upup/pkg/kutil/kubecfg_builder.go]: I don't think I do @kris-nova ... Is this because we should be writing to a stream somewhere?

**Pull request merged**